pybliometrics/scopus/tests/test_ScopusSearch.py

```diff
@@ -18,6 +18,7 @@
 # Set to refresh=False because of citation count
 s_au = ScopusSearch('AU-ID(24320488600)', refresh=30)
 s_j = ScopusSearch('SOURCE-ID(22900) AND PUBYEAR IS 2010', refresh=30)
+s_d = ScopusSearch("DOI(10.1038/s41556-022-01034-3)", unescape=True, refresh=30)
```

Comment:
Don't forget to turn the test around when the default value for `unescape` changes.

Reply:
Since the test already had the new default value nothing changes right? Or you mean that I should test the other case `unescape=False`

Reply:
Yes

Reply:
Yes, test with the non-default case.